fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -589,7 +625,14 @@ void testDataLoad() {
                     .setText(randomWords[1])
                     .setGroup(2)
                     .build();
-            recordStore.saveRecord(dylan);
+            try {
+                recordStore.saveRecord(dylan);
+            } catch (Exception ex) {
+                System.out.println("***********************************Index number****************************************");
+                System.out.println("i=" + String.valueOf(i));
+                System.out.println("***************************************************************************************");
```

Comment:
Is this supposed to remain in the final version? It's only a test, but it still seems a little loud.

Reply:
sorry this is debugging code kept here accidentally, will be removed

Reply:
Removed it